README.md

```diff
@@ -120,6 +120,7 @@ Name|Meeting Time|Meeting Notes|Discussions|[Technical Committee](./community-me
 Specification: General|Every Tuesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo/edit)|[Slack](https://cloud-native.slack.com/archives/C01N7PP1THC)
 Specification: Logs|Every Wednesday at 10:00 PT|[Google Doc](https://docs.google.com/document/d/1cX5fWXyWqVVzYHSFUymYUfWxUK5hT97gc23w595LmdM/edit#)|[Slack](https://cloud-native.slack.com/archives/C01N5UCHTEH)|[Tigran Najaryan](https://github.com/tigrannajaryan)
 Specification: Sampling|Every Thursday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/1gASMhmxNt9qCa8czEMheGlUW2xpORiYoD7dBD7aNtbQ/)|[Slack](https://cloud-native.slack.com/archives/C027DS6GZD3)|[Josh MacDonald](https://github.com/jmacd)
+Specification: Configuration|Every other Friday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1uNgWQLQZcSVfBLRXfW9XCVJmKa5NH9R15fSOXhmpGWw/edit)|[Slack](https://cloud-native.slack.com/archives/C01N7PP1THC)|[Carlos Alberto](https://github.com/carlosalberto)
```

Comment:
To confirm: is this Google Doc owned by Otel org?

Reply:
yes